screens/popups/transfer_pokemon/transfer.gui_script

```diff
@@ -326,13 +326,13 @@ function on_input(self, action_id, action)
 		if self.move_button_enabled then
 			a = gooey.button("btn_move", action_id, action, function() move(self) end, function(b) gooey_buttons.common_button(b, gui.get_node("txt_move")) end)
 			c = gooey.button("btn_release", action_id, action, function() release(self) end)
-			d = gooey.button("btn_share", action_id, action, function() share.export(self.pokemon_id) end, gooey_buttons.share)
 		end
+		d = gooey.button("btn_share", action_id, action, function() share.export(self.pokemon_id) end, gooey_buttons.share)
```

Comment:
This change appears to let you share even if you cannot move the Pokémon, is that correct?